fields-change/news-field-change.service.ts

```diff
@@ -0,0 +1,27 @@
+import { FieldsChangeService, FieldWrapper, FIELDS_CHANGE_SERVICE_TOKEN } from "progress-sitefinity-adminapp-sdk/app/api/v1";
+import { Injectable } from "@angular/core";
+
+const NEWS_TYPE_FULL_NAME = "Telerik.Sitefinity.News.Model.NewsItem";
+const TITLE_FIELD_NAME = "Title";
+const RITCH_TEXT_EDITOR_FIELD_NAME = "Content";
+
+@Injectable()
+export class NewsFieldsChangeService implements FieldsChangeService {
```

Comment:
name it mb something like CustomFieldsChangeService?

Reply:
and leave js doc around here which is which and what it does

Reply:
But it's not for custom fields. It's for all fields in the form.

Reply:
agree with Marto here. Custom shows that the service is custom and not that it is related to custom fields